### PR TITLE
5896 create ui home cate

### PR DESCRIPTION
**Pull request by Namvn97**
https://edu-redmine.framgia.vn/issues/5896
create UI categories screen home

### CODE REVIEW

**Comment by Namvn97** [comment]
ready

File: app/src/main/java/framgia/phannam/android/nmusic/ui/home/FragmentCategories.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_gategories, container, false);
+        mContext = getContext();
+        mRecyclerView = view.findViewById(R.id.recyler_categories);
+        GridLayoutManager layoutManager = new GridLayoutManager(getActivity(), SPAN_COUNT);
+        mRecyclerView.setLayoutManager(layoutManager);
+        mCategoriesAdapter = new HomeCategoriesAdapter(null,mContext);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/home/FragmentCategories.java]: format code. ctrl + alt + l & ctrl + alt + o tất cả các class.

File: app/src/main/java/framgia/phannam/android/nmusic/ui/home/FragmentCategories.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        mCategoriesAdapter.setOnGenreClickListener(this);
+        mRecyclerView.setAdapter(mCategoriesAdapter);
+        mPresenter = new HomePresenter(this,
+                CatesRepository.getInstance(new CategoriesLocalDataSource()));
+        mPresenter.loadCategories();
+        return view;
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/framgia/phannam/android/nmusic/ui/home/FragmentCategories.java]: remove bl

**Comment by Namvn97** [comment]
ready

**Pull request merged**